src/Type.cc

```diff
@@ -2690,7 +2695,21 @@ TypePtr merge_type_list(detail::ListExpr* elements)
 		return t;
 
 	for ( size_t i = 1; t && i < tl.size(); ++i )
-		t = merge_types(t, tl[i]);
+		{
+		auto& tl_i = tl[i];
```

Comment:
```suggestion
		const auto& tl_i = tl[i];
```

Reply:
Okay I've added that tweak and made `merge_types()` externally available again.